templates/vsc/js/basic-custom-engine-agent/.localConfigs.playground.tpl

```diff
@@ -9,4 +9,4 @@ AZURE_OPENAI_API_KEY=
 AZURE_OPENAI_ENDPOINT=
 AZURE_OPENAI_DEPLOYMENT_NAME=
 {{/useAzureOpenAI}}
-TEAMSFX_NOTIFICATION_STORE_FILENAME=.notification.testtoolstore.json
+
```

Comment:
Isn't this used anymore? Add @qinzhouxu to confirm here

Reply:
This should be used by TeamsFx SDK for notification local storage.

Reply:
There is no reference for this variable in Playground now.